ament_cmake_python/cmake/ament_get_python_install_dir.cmake

```diff
@@ -0,0 +1,29 @@
+# Copyright 2014 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Get the Python installation directory.
+#
+# :param python_install_dir_out: this var will be populated with the path to the
+#   Python installation directory.
+# :type package_name: string
+#
+macro(ament_get_python_install_dir python_install_dir_out)
```

Comment:
@MaximilienNaveau why not a `function`, as @sloretz suggested?

Reply:
Because for now all the public "function" defined in this package are implemented as macros:
- https://github.com/ament/ament_cmake/blob/master/ament_cmake_python/cmake/ament_python_install_module.cmake
- https://github.com/ament/ament_cmake/blob/master/ament_cmake_python/cmake/ament_python_install_package.cmake
The `function` is used with an underscore (`function(_function_name args)`) which indicates them as private in this package.

I can still use a `function` if you really want to.

Reply:
Note that none of those macros pollutes the calling scope. I'd recommend using a `function` for this case.

Reply:
I did not see that, I did the modification.